federation/src/graphql/api/1_1/resolver/TestResolver.ts

```diff
@@ -0,0 +1,21 @@
+import { Field, ObjectType, Query, Resolver } from 'type-graphql'
+import { federationLogger as logger } from '@/server/logger'
+
+@ObjectType()
+class GetTestApiResult {
+  constructor(apiVersion: string) {
+    this.api = `${apiVersion}`
+  }
+
+  @Field(() => String)
+  api: string
+}
```

Comment:
Yes for sure, but this Resolver is only an interims class for research how to implement multiple graphql-endpoint support base on api-versions.
in the next step of federation it will be exchanged with a resolver of a real business endpoint.